client/src/Views/Home/Home.js

```diff
@@ -1,15 +1,41 @@
-import React, { Component } from 'react';
-import { Typography } from '@material-ui/core';
+import React, { Component } from "react";
+import { Typography, Grid } from "@material-ui/core";
+import StyledButton from "Components/Button/StyledButton";
+import AuthUserContext from "Components/Session/AuthUserContext";
+import withAuthorization from "Components/Session/withAuthorization";
 
 class Home extends Component {
-  state = {  }
+  state = {};
+
+  logOut = () => {
+    this.context.setToken(null);
+    this.context.setUserId(null);
+  };
```

Comment:
Yeah, I think this function should be in the context itself, and then called here

Reply:
Done